modules/charm.lua

```diff
@@ -321,10 +340,34 @@ function Module:Render()
 					ImGui.TableNextColumn()
 					ImGui.Text(string.format("%s", data.name))
 					ImGui.TableNextColumn()
+					ImGui.Text(string.format("%s", data.lvl))
```

Comment:
fyi ImGui.Text has string.format built in, so you can just remove the string.format part.